### PR TITLE
Assorted small fixes for CockroachDB tests

### PR DESCRIPTION
**Pull request by otan**

### CODE REVIEW

**Comment by otan** [comment]
cc @tomhoule

File: introspection-engine/introspection-engine-tests/tests/commenting_out/mod.rs [hunk 1]
```diff
@@ -197,6 +197,55 @@ async fn unsupported_type_keeps_its_usages(api: &TestApi) -> TestResult {
     Ok(())
 }
 
+
+#[test_connector(tags(Postgres))]
```
Comment (tomhoule) [inline introspection-engine/introspection-engine-tests/tests/commenting_out/mod.rs:201]: Did you mean `tags(Cockroach)` here?

**Review by tomhoule — COMMENTED** [review]
One typo I think. Ping @do4gr to validate the approach.

File: introspection-engine/introspection-engine-tests/tests/commenting_out/mod.rs (hunk 1 above)
Comment (otan) [reply introspection-engine/introspection-engine-tests/tests/commenting_out/mod.rs]: yep! fixed.
funny it passed for postgres too tho.

**Review by do4gr — APPROVED** [review]

**Pull request merged**